services/src/main/scala/cromwell/services/metadata/impl/MetadataDatabaseAccess.scala

```diff
@@ -167,10 +167,14 @@ trait MetadataDatabaseAccess {
     metadataDatabaseInterface.getWorkflowLabels(id.toString)
   }
 
-  def workflowExistsWithId(possibleWorkflowId: String)(implicit ec: ExecutionContext): Future[Boolean] = {
+  def workflowWithIdExistsInMetadata(possibleWorkflowId: String)(implicit ec: ExecutionContext): Future[Boolean] = {
```

Comment:
ToL Remove this and `workflowWithIdExistsInMetadataSummaries` as they don't encapsulate any logic.